packages/blip/app/components/patient-data.js

```diff
@@ -217,7 +217,7 @@ class PatientDataPage extends React.Component {
         loader = <Loader />;
         break;
       default:
-        errorDisplay = <p>{errorMessage ?? t('Failed somewhere')}</p>;
+        errorDisplay = <p id="display-message">{errorMessage ?? t('Failed somewhere')}</p>;
```

Comment:
```suggestion
        errorDisplay = <p id="loading-error-message">{errorMessage ?? t('Failed somewhere')}</p>;
```

Reply:
I don't see the benefit of specializing so much this kind of id

Reply:
Just to be sure it is an unique id. If too generic, we may ends up having 2x the same id at the same time.